packages/css/src/plugins/__tests__/sort-shorthand-declarations.test.ts

```diff
@@ -386,13 +388,115 @@ describe('sort shorthand vs. longhand declarations', () => {
+    const actual = transform(outdent`
+      ._oqicidpf{padding-top:0}
+      ._1rmjidpf{padding-right:0}
+      ._cjbtidpf{padding-bottom:0}
+      ._pnmbidpf{padding-left:0}
+      ._glte7vkz{width:1pc}
+      ._165t7vkz{height:1pc}
+      ._ue5g1408{margin:0 var(--ds-space-800,4pc)}
+      ._1yag1dzv{padding:var(--ds-space-100) var(--ds-space-150)}
+      ._dbjg12x7{margin-top:var(--ds-space-075,6px)}
+
+      @media (min-width:1200px){
+        ._jvpg11p5{display:grid}
+        ._szna1wug{margin-top:auto}
+        ._13on1wug{margin-right:auto}
+        ._1f3k1wug{margin-bottom:auto}
+        ._inid1wug{margin-left:auto}
+        ._1oqj1epz{padding:var(--ds-space-1000,5pc)}
+        ._12wp9ac1{max-width:1400px}
+        ._jvpgglyw{display:none}
+      }
+    `);
+
+    expect(actual).toMatchInlineSnapshot(`
+      "
+      ._ue5g1408{margin:0 var(--ds-space-800,4pc)}
+      ._1yag1dzv{padding:var(--ds-space-100) var(--ds-space-150)}._oqicidpf{padding-top:0}
+      ._1rmjidpf{padding-right:0}
+      ._cjbtidpf{padding-bottom:0}
+      ._pnmbidpf{padding-left:0}
+      ._glte7vkz{width:1pc}
+      ._165t7vkz{height:1pc}
+      ._dbjg12x7{margin-top:var(--ds-space-075,6px)}
+
+      @media (min-width:1200px){
+        ._1oqj1epz{padding:var(--ds-space-1000,5pc)}
+        ._jvpg11p5{display:grid}
+        ._szna1wug{margin-top:auto}
+        ._13on1wug{margin-right:auto}
+        ._1f3k1wug{margin-bottom:auto}
+        ._inid1wug{margin-left:auto}
+        ._12wp9ac1{max-width:1400px}
+        ._jvpgglyw{display:none}
+      }"
+    `);
+  });
+
+  it('sorts border, border-top, border-top-color', () => {
+    const actual = transform(outdent`
+
+      ._abcd1234 { border-top-color: red }
+      ._abcd1234 { border-top: 1px solid }
+      ._abcd1234 { border: none }
+      ._abcd1234:hover { border-top-color: red }
+      ._abcd1234:hover { border-top: 1px solid }
+      ._abcd1234:hover { border: none }
+      ._abcd1234:active { border-top-color: red }
+      ._abcd1234:active { border-top: 1px solid }
+      ._abcd1234:active { border: none }
+      @supports (border: none) {
+        ._abcd1234 { border-top-color: red }
+        ._abcd1234 { border-top: 1px solid }
+        ._abcd1234 { border: none }
+      }
+      @media (max-width: 50px) { ._abcd1234 { border-top-color: red } }
+      @media (max-width: 100px) { ._abcd1234 { border-top: 1px solid } }
+      @media (max-width: 120px) {
+        ._abcd1234 { border-top-color: red }
+        ._abcd1234 { border-top: 1px solid }
+        ._abcd1234 { border: none }
+      }
+      @media (max-width: 150px) { ._abcd1234 { border: none } }
+    `);
+
+    expect(actual).toMatchInlineSnapshot(`
+      "
+      ._abcd1234 { border: none }
+      ._abcd1234 { border-top: 1px solid }
+      ._abcd1234 { border-top-color: red }
+      ._abcd1234:hover { border: none }
+      ._abcd1234:hover { border-top: 1px solid }
+      ._abcd1234:hover { border-top-color: red }
```

Comment:
Oh is this fixed now? 😮

Reply:
yeah i fixed it in https://github.com/atlassian-labs/compiled/pull/1730/commits/18adcbe2bd8fe30f3cb1deebf7db9ea6e260e4ee

Reply:
whoops not that commit

Reply:
fixed in this one https://github.com/atlassian-labs/compiled/pull/1730/commits/8b6edba726e3733b36a0c286bd74f0eb31539f50